onnx/checker.h

```diff
@@ -43,12 +49,25 @@ class CheckerContext {
   explicit CheckerContext() : ir_version(-1) {}
 };
 
+struct LexicalScopeContext {
+  std::unordered_set<std::string> output_names;
```

Comment:
nit: maybe just "names"?

Reply:
I named this output_names to be consistent with the code in the checker. maybe we should change both?